src/status_im/contexts/communities/events.cljs

```diff
@@ -25,7 +25,7 @@
                   token-permissions-check id last-opened-at]
            :as   community}       (data-store.communities/<-rpc community-js)
           previous-last-opened-at (get-in db [:communities id :last-opened-at])]
-      (when (>= clock (get-in db [:communities id :clock]))
+      (when (and (>= clock (get-in db [:communities id :clock])) id)
```

Comment:
id should be first

Reply:
Thank you @flexsurfer for reviewing PR. Moved id to beginning.

( I had this change in mind 😅, and made it, but apparently in wrong branch https://github.com/status-im/status-mobile/commit/9d1154d39e98c030eb19a8cb76de388f596751af#diff-63b184880cbf5c0450f717326801b4e79c54135e30c0bdffc1528b4c8fd89b65 )